static/coffee/controllers.coffee

```diff
@@ -485,7 +485,7 @@ controllers.controller('HomeController', ['$scope', '$controller', 'LabelService
         $scope.labels = labels
       )
     else if tab == 'users'
-      UserService.fetchNonPartner(true).then((users) ->
+      UserService.fetchAll(true).then((users) ->
```

Comment:
@rowanseymour We have now added a select/dropdown for filtering and the partner name is already included in the display:

![org users](https://cloud.githubusercontent.com/assets/3963913/18133314/66570352-6f9a-11e6-9aa7-c7d5a23d5f03.png)

![select filter](https://cloud.githubusercontent.com/assets/3963913/18133325/6dbd1730-6f9a-11e6-984d-e2f7f9c5852b.png)